webapp-django/crashstats/monitoring/tests/test_views.py

```diff
@@ -278,3 +280,42 @@ def fake_es_instance(**config):
         )
 
         eq_(es_instance.info.call_count, 2)
+
+    def test_assert_supersearch_counts(self):
+
+        searches = []
+
+        def mocked_supersearch_get(**params):
+            searches.append(params)
+            eq_(params['product'], [settings.DEFAULT_PRODUCT])
+            # print params
+            if len(searches) == 1:
+                # this is the first one
+                eq_(params['_results_number'], 1)
+                eq_(params['_columns'], ['uuid'])
+                return {
+                    'hits': [
+                        {'uuid': '12345'},
+                    ],
+                    'facets': [],
+                    'total': 30002
+                }
+            else:
+                # second search
+                eq_(params['_results_number'], 100)
+                eq_(params['_results_offset'], 30000)
+                return {
+                    'hits': [
+                        {'uuid': '0128940'},
+                        {'uuid': '9156826'},
+                        {'uuid': '3969175'},
+                    ],
+                    'facets': [],
+                    'total': 30004
+                }
+
+        SuperSearch.implementation().get.side_effect = (
+            mocked_supersearch_get
+        )
+        assert_supersearch_counts()
```

Comment:
My bad, I read ``return`` instead of ``assert`` in the function. Still, I would have expected to see a test for when the check fails.